spread.yaml

```diff
@@ -14,8 +14,13 @@ backends:
         systems:
             - ubuntu-16.04-64-grub
             - ubuntu-16.04-32-grub
+    qemu:
+        systems:
+            - ubuntu-16.04:
+                username: ubuntu
+                password: ubuntu
 
-path: $[PROJECT_PATH]
+path: /home/gopath/src/github.com/snapcore/snapd
```

Comment:
That is what I had initially, but it errors with:
`error: remote project path must be absolute and not /: $(PROJECT_PATH)`